cmd/awscollector/build_docker_binary.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+BINARY=aoc_linux_x86_64
+if [[ $TARGETPLATFORM == "linux/arm64" ]]; then
+  BINARY=aoc_linux_aarch64
+fi
+echo binary name is ${BINARY}
+if [[ -f /workspace/${BINARY} ]]; then
+  echo Copying binary &&
+    mkdir -p /workspace/build/linux &&
+    cp /workspace/${BINARY} /workspace/build/linux/aoc_linux
+else
+  echo Building binary &&
+    go mod download &&
+    make amd64-build &&
```

Comment:
So the assumption we make here is if one of the binaries do not exist, we will build all the binaries right?

Reply:
Yes

Reply:
After speaking with @Aneurysm9 we should not build the binary inside the docker container since make will build the binary and in ci we cache the binary.